Using the GNU standardised $DESTDIR Makefile variable instead of $ROOTPATH.
The install: step probably need to be reworked before being ported back to the master branch.
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,12 @@
 PACKAGE=perfsonar-lsregistrationdaemon
-ROOTPATH=/usr/lib/perfsonar
+DESTDIR?=/usr/lib/perfsonar
 CONFIGPATH=/etc/perfsonar
 VERSION=3.5.1
 RELEASE=0.0.a1
+BIN=$(DESTDIR)/bin
+DOC=$(DESTDIR)/doc
+ETC=$(DESTDIR)/etc
+LIB=$(DESTDIR)/lib
 
 default:
 	@echo No need to build the package. Just run \"make install\"
@@ -17,9 +21,8 @@
 	rm -rf /tmp/$(PACKAGE)-$(VERSION).$(RELEASE)
 
 install:
-	mkdir -p ${ROOTPATH}
-	tar ch --exclude=etc/* --exclude=*spec --exclude=dependencies --exclude=MANIFEST --exclude=Makefile -T MANIFEST | tar x -C ${ROOTPATH}
-	for i in `cat MANIFEST | grep ^etc/ | sed "s/^etc\///"`; do  mkdir -p `dirname $(CONFIGPATH)/$${i}`; if [ -e $(CONFIGPATH)/$${i} ]; then install -m 640 -c etc/$${i} $(CONFIGPATH)/$${i}.new; else install -m 640 -c etc/$${i} $(CONFIGPATH)/$${i}; fi; done
+	install -d $(BIN) $(DOC) $(ETC) $(LIB)
+	tar ch ./lib/perfSONAR_PS/LSRegistrationDaemon | tar x -C $(LIB)
 
 test:
 	PERL_DL_NONLAZY=1 /usr/bin/perl "-MExtUtils::Command::MM" "-e" "test_harness(0)" t/*.t
--- a/perfsonar-lsregistrationdaemon.spec
+++ b/perfsonar-lsregistrationdaemon.spec
@@ -72,7 +72,7 @@
 %install
 rm -rf %{buildroot}
 
-make ROOTPATH=%{buildroot}/%{install_base} CONFIGPATH=%{buildroot}/%{config_base} install
+make DESTDIR=%{buildroot}/%{install_base} CONFIGPATH=%{buildroot}/%{config_base} install
 
 mkdir -p %{buildroot}/etc/init.d
 
